src/test/java/nextstep/subway/member/MemberAcceptanceTest.java

```diff
@@ -22,7 +22,7 @@
 
     @DisplayName("Feature : 회원 정보를 관리한다.")
     @Nested
-    class Feature_ManageMbmer{
+    class Feature_ManageMember{
```

Comment:
자바 컨벤션으로 많이 사용되고 있는 카멜 컨벤션으로 맞춰주면 어떨까요?

```suggestion
    class FeatureManageMember{
```